/armor_node:
  ros__parameters:
    h_max: 255
    h_min: 0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    s_max: 30
    s_min: 0
    use_sim_time: false
    v_max: 50
    v_min: 0

